src/main/java/com/sfuentes/Worker.java

```diff
@@ -0,0 +1,23 @@
+package com.sfuentes;
+
+public class Worker {
+
+  int workerId;
+  int hours;
+
+  public Worker(int workerId, int hours) {
+    this.workerId = workerId;
+    this.hours = hours;
+  }
+
+  public double calculateSalary() {
+
+    if (hours <= 40)
+      return hours * 200;
+    else {
+      hours -= 40;
```

Comment:
Here you're changing the state of this object by removing 40 hours. It would be less intrusive to just calculate the extra hours when you need it without modifying the object's state.
Something like this:
return 8_000 + (hours-40)* 250;

Or even more clear, to add a new property for the rate(which is the amount of money per hour)

Reply:
fixed.